adaptdl_cli/adaptdl_cli/registry.py

```diff
@@ -75,15 +93,14 @@ def _fix_daemon_json():
 
 
 def registry_running():
-    svc = json.loads(subprocess.check_output(['kubectl', 'get', 'svc', '-l',
-                                             'app=docker-registry', '-ojson']))
+    svc = json.loads(subprocess.check_output(['kubectl', 'get', 'svc',
```

Comment:
Prefer to use the `kubernetes` dependency package instead of calling `kubectl` in a subprocess. This is the convention elsewhere in the CLI, since we have control over the `kubernetes` package version but not the `kubectl` version.

Reply:
Sure. However I found `kubectl` usage at multiple places in the cli e.g in `cp`, `logs`, `ls` etc. We are entirely not independent of `kubectl`.